src/Sidekick/Helpers/Input/EventsHandler.cs

```diff
@@ -237,6 +242,17 @@ private static void TriggerHideout()
             SendKeys.SendWait(Input.KeyCommands.HIDEOUT);
         }
 
+        public static async void TriggerOpenSearch()
+        {
+            Legacy.Logger.Log("Hotkey for Hideout triggered.");
```

Comment:
`"Hotkey for Hideout triggered."` -> `"Hotkey for opening search query in browser triggered."`